README.md

```diff
@@ -95,36 +95,22 @@ To get a local copy up and running, follow these steps.
 ### Prerequisites
 
 In order to run this project you need:
+- browser
+- code editor
+- Npm
 
 
-### Setup
-
-Clone this repository to your desired folder:
-
-<!--
-Example commands:
-
-```sh
```

Comment:
Great work adding a readme file, i think it would be more professional if you use the template that we were given to write your readme.[link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)